tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## Version 0.1.0 (06-26-2023)
```

Comment:
Should this be in YYYY-MM-DD format?

Reply:
sorry I copied over from another tool changelog, yes will fix to match python :)